runtime/doc/treesitter.txt

```diff
@@ -573,6 +573,15 @@ associated with patterns:
     • `injection.parent` - indicates that the captured node’s text should
       be parsed with the same language as the node's parent LanguageTree.
 
+Injection queries are currently run over the entire buffer, which can be slow
+for large buffers. To disable injections for a certain language, just place an
+empty query file in your 'runtimepath'. For example, to disable injections for
+`c`, use the following `queries/c/injections.scm`: >query
+
+    ; This file contains no captures (only comments), thus injections are
+    ; ignored for this language.
+    ; Note that these comments are arbitrary and not necessary.
+
```

Comment:
```suggestion
Injection queries are currently run over the entire buffer, which can be slow
for large buffers. To disable injections for, e.g.,  `c`, just place an
empty `queries/c/injections.scm` file in your 'runtimepath'.
```

Reply:
But wouldn't it also be helpful to know that the file can be either empty, or contain only comments?

Reply:
At this point, people only need to know how to disable injections, and the simplest way will do. (It just has to be correct, not complete.)

Reply:
(and I think if the comment happens to be `; extends`, then it _won't_ disable anything -- and now things become too complicated to explain in full)

Reply:
Sure. I still feel like I would personally get confused by an empty file and think it was a mistake, thus I would like  a comment explaining its purpose, but I guess the user could fiddle around for 30 seconds and figure things out